paladin/mod.ts

```diff
@@ -0,0 +1,186 @@
+import { Drash } from "../deps.ts";
```

Comment:
deno fmt this file

Reply:
https://github.com/drashland/deno-drash-middleware/pull/25/commits/be604ab3fce854d077200672fc0fc35d77ab7402